chapter_computational-performance/multiple-gpus.md

```diff
@@ -159,7 +159,7 @@ print('b1 weight:', new_params[1])
 print('b1 grad:', new_params[1].grad)
 ```
 
-由于我们还没有进行任何计算，偏置参数的梯度仍然为零。现在假设有一个向量分布在多个GPU上。下面的[**`allreduce`函数将所有向量相加，并将结果广播回所有GPU**]。请注意，要使其工作，我们需要将数据复制到累积结果的设备。
```

Comment:
Hey @zhuyuanxiang , could you recover the `[**`, `**]`, `(**`, `**)` symbols back? We are using these symbols for building slides (https://book.d2l.ai/user/slides.html). Thanks!

Reply:
Hey @goldmermaid, I have recovered the `[**` and `**]` symbols back.

Reply:
Thks for your explanation.